addons/chatmon/readme.md

```diff
@@ -21,28 +21,62 @@
 #### Tigger Fields:
 * `from`: a set of chat name you would like the trigger to happen on, the full list of chat modes can be found here [resources/chat.lua](https://github.com/Windower/Resources/blob/master/resources_data/chat.lua)
 
-  * in addition to the stated chat modes, you can provide `"all"` this flag will trigger checks of all text that is displayed in the chat.
+  * in addition to the stated chat modes, you can provide `all`.
+    * `all` will trigger all that is displayed in the chat.
+    * `all` respect the in-game blacklist, the other channel options do not.
+    * `all` is affected unpredictabilty by addons which alter the text, just be aware when making your match text. ex addon: battlemod
+    * `all` does contain the full text displayed in the chat, meaning the sender's name will be part of the beginning of the text that is evaluated.
 
 * `notFrom`: a set of chat mode names you want the tigger to **ignore**.
 
 * `match`: text you whould like to match to have the trigger sound to be played, 
 
   * `*` can be used as a wild card. Ex. `*Nif*` will trigger for any text containing with `Nif` such as `Hi Nifim` or `Hi Niflheim` or `lolNifim`
 
-  * `|` cab be used to seperate multiple words you want to tigger for such as `Jo|Yo`, will trigger `Jo` and `Yo`
+  * `|` can be used to seperate multiple words you want to tigger for such as `Jo|Yo`, will trigger `Jo` and `Yo`
 
   * `<name>` can be used as a special flag to cover your character name in a number of common contexts.
 
-  * when the trigger source is `tell`, `invite`, or `examine` the text evaluated for match is the **senders name** all other sources evaluate the message text.
+  * when the trigger source is `tell`, `emotes`, `invite`, or `examine` the text evaluated for match is the **senders name** all other sources evaluate the message text.
```

Comment:
Er, why is `tell` in that list? Should `tell` match the text, not the name? Applies to the below comments as well. Or is that plugin behavior we're trying to match?

Also, did we settle on `emotes` instead of `emote`? I probably prefer the latter 🤔 But open to suggestions.